### DISCUSSION TITLE
"image" prepended when doing a build

### DISCUSSION THREAD

**Discussion by magick93** [maizzle/framework#1335] [Q&A]
Hello

I have the following code in a template:

```
<a href="@{{ .ConfirmationURL }}"
                class="inline-block sm:block py-4 px-8 rounded shadow text-sm/4 text-center font-bold text-white bg-brand-500 hover:bg-brand-600">
                <!--[if mso]><i style="letter-spacing: 32px; mso-font-width: -100%; mso-text-raise:30px;">&#8202;</i><![endif]-->
                <span class="mso-text-raise-4">Verify</span>
                <!--[if mso]><i style="letter-spacing: 32px; mso-font-width: -100%;">&#8202;</i><![endif]-->
              </a>
```

When I do a build, it generates the following:

```
<a href="images/{{ .ConfirmationURL }}" class="sm-block hover-bg-brand-600" style="text-decoration: none; display: inline-block; border-radius: 4px; background-color: #0052E2; padding: 16px 32px; text-align: center; font-size: 14px; line-height: 16px; font-weight: 700; color: #fff; box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.1), 0 1px 2px 0 rgba(0, 0, 0, 0.06)">
                  <!--[if mso]><i style="letter-spacing: 32px; mso-font-width: -100%; mso-text-raise:30px;">&#8202;</i><![endif]-->
                  <span style="mso-text-raise: 16px">Verify</span>
                  <!--[if mso]><i style="letter-spacing: 32px; mso-font-width: -100%;">&#8202;</i><![endif]-->
                </a>
```

I'm wondering why the `images/` is added to the `href`? 

If I just want to have `{{ .ConfirmationURL }}` as the href url, what is the correct notation?

Thanks

**Comment by cossssmin** [comment]
Hi there, are you perhaps using the Maizzle 5 beta?

This behavior was reverted in [v5.0.0-beta.18](https://github.com/maizzle/framework/releases/tag/v5.0.0-beta.18), it was prepending the `build.assets.destination` path to href/src if it wasn't set in the config.

In case you're not using the v5 beta, make sure to specify which tags the `baseURL` should apply to:

https://maizzle.com/docs/transformers/base-url#tags

**Reply by magick93** [reply]
Thanks. 

I'm using `latest`. 

Even after changing to the following config, I was still getting the `images/` appended.

```
build: {
    templates: {
      source: 'src/templates',
      destination: {
        path: 'dist',
      },
    },
    assets: {
      source: 'src/images',
      destination: '',
    },
  },
  ```
  
  Once I changed `baseURL` to `''` the problem was solved. 
  
  Thanks again.
